framework/core-compose/src/commonMain/kotlin/com/cramsan/framework/core/compose/ui/ThemePreviewer.kt

```diff
@@ -0,0 +1,157 @@
+package com.cramsan.framework.core.compose.ui
+
```

Comment:
This file creates some functions to render JVM previews for the colors, typography and shapes as configured by the current theme. These functions are intended to be called by the applications to be able to render the previews for their respective theme.